Rymera-Web-Co/wcvendors
# Fixed: Vendor Membership being logged in the Commissions table #817

### All Submissions:






### Changes proposed in this Pull Request:


- Check if the order has vendors.
- Format some code

Closes #817.

### How to test the changes in this Pull Request:

1. Create a vendor's membership
2. Purchase the vendor's membership as a vendor
3. Check the commissions' table, and not see the vendor's membership purchase is logged

### Other information:
